common/util.go

```diff
@@ -380,3 +380,10 @@ func RatPriceInfo(priceInfo *net.PriceInfo) (*big.Rat, error) {
 
 	return big.NewRat(priceInfo.PricePerUnit, pixelsPerUnit), nil
 }
+
+func JoinPaths(path1, path2 string) string {
```

Comment:
Would `JoinURL(url, path string)` be a more descriptive name if this function is only intended to be used as an alternative to `path.Join()` when the first string param is a URL?

Reply:
Fixed